near-sdk-core/src/info_extractor/attr_sig_info.rs

```diff
@@ -97,13 +96,36 @@ impl AttrSigInfo {
             }
         }
 
+        let is_view = if let Some(ref receiver) = receiver {
```

Comment:
According to this code static method (the one that does not use `self`) that is not decorated with `#[init]` is not a view method, which is wrong.

Reply:
How so? Else branch executes if receiver is None and if `is_init` is false `is_view` sets to true.

Reply:
You are right.